internal/main.go

```diff
@@ -82,7 +85,10 @@ func main() {
 
 	if flags.clearCache {
 		if err := os.Remove(flags.configCache); err != nil {
-			logger.Err("unable to clear cache: %v", err)
+			logger.Err("unable to clear config cache: %v", err)
+		}
+		if err := os.Remove(flags.stateCache); err != nil {
```

Comment:
That's fair.  Actually, the config cache is no longer a cache either, since the fetch stages are the only ones that can create it, but I guess we have a compatibility constraint there.

Reply:
Decided to go the other direction: https://github.com/coreos/ignition/pull/1251